### PR TITLE
Include test case for clipboard for the 'secrethub generate' command

### PR DESCRIPTION
**Pull request by edif2008**
This PR contains one extra test in which the clipboard is selected as the output

### CODE REVIEW

File: internals/secrethub/generate_test.go [hunk 1]
```diff
@@ -46,6 +47,24 @@ func TestGenerateSecretCommand_run(t *testing.T) {
...
+				copyToClipboard: true,
+				clipper:         fakeclip.New(),
+			},
+			writeFunc: func(path string, data []byte) (*api.SecretVersion, error) {
+				return &api.SecretVersion{Version: 1}, nil
+			},
+			path: "namespace/repo/secret",
+			data: []byte("random generated secret"),
+			out: "A randomly generated secret has been written to namespace/repo/secret:1.\n" +
+				"The generated value has been copied to the clipboard. It will be cleared after Less than a second.\n",
```
Comment (Marton6) [inline internals/secrethub/generate_test.go:66]: Could we also check whether the value has actually been written to the fake clipboard?

Comment (edif2008) [reply internals/secrethub/generate_test.go:71]: Done

File: internals/secrethub/generate_test.go [hunk 2]
```diff
@@ -184,17 +221,16 @@ func TestGenerateSecretCommand_run(t *testing.T) {
 				}, tc.clientCreationErr
 			}
 
-			io := fakeui.NewIO(t)
-			tc.cmd.io = io
-
 			// Act
 			err := tc.cmd.run()
+			resClip, _ := tc.cmd.clipper.ReadAll()
```
Comment (Marton6) [inline internals/secrethub/generate_test.go:226]: After this line you could call `assert.OK` on the error value, to make sure it is always nil.

Comment (edif2008) [reply internals/secrethub/generate_test.go:226]: Good to know. 
Done

File: internals/secrethub/generate_test.go [hunk 3]
```diff
@@ -20,111 +21,141 @@ func newIntValue(v int) intValue {
 
 func TestGenerateSecretCommand_run(t *testing.T) {
 	testErr := errio.Namespace("test").Code("test").Error("test error")
+	testData := []byte("random generated secret")
+	testPath := "namespace/repo/secret"
 
 	cases := map[string]struct {
 		cmd               GenerateSecretCommand
 		writeFunc         func(path string, data []byte) (*api.SecretVersion, error)
 		clientCreationErr error
```
Comment (Marton6) [inline internals/secrethub/generate_test.go:30]: This should be named `newClientErr` so it is consistent with the naming in other tests.

Comment (edif2008) [reply internals/secrethub/generate_test.go:30]: Fixed

**Pull request merged**